PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -1685,13 +1685,9 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 				case IDD_FINDINFILES_FIND_BUTTON:
 				{
 					setStatusbarMessage(TEXT(""), FSNoMessage);
-					const int filterSize = 256;

```

Comment:
Modifying the value from 256 to 1024 is enough fairly for the filter size.

Reply:
OK.  I can do that tomorrow if you are sure you don't want a dynamic size.  I think there are probably 3 places it should be changed.

Reply:
And you really need to get this out of "combination" with 11988!

Reply:
@alankilborn 
Sorry I don't follow you. Could you elaborate it?

Reply:
@alankilborn OK, I see.
Yes, it's confusing. It's due to the wrong number in the previous commit.

Reply:
So just to clarify, remove the function that reads a variable size string and just bump the buffer size to 1024?

Reply:
Yes. Just modify `const int filterSize = 256;` to const int filterSize = 1024; from the initial state of code source.
The added function is not necessary.

Reply:
@donho It should match what you suggested now.